### PR TITLE
fix(accountsdb): inf loop

### PR DESCRIPTION
**Pull request by 0xNineteen**

### CODE REVIEW

**Comment by 0xNineteen** [comment]
* **#383** <a href="https://app.graphite.dev/github/pr/Syndica/sig/383?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @0xNineteen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by InKryption — APPROVED** [review]
lgtm

File: src/accountsdb/index.zig [hunk 1]
```diff
@@ -230,14 +230,7 @@ pub const AccountIndex = struct {
...
         // traverse until you find the end
         var curr_ref = map_entry.value_ptr.ref_ptr;
-        if (@import("builtin").mode == .Debug and curr_ref.slot == account_ref.slot) {
-            std.debug.panic("duplicate slot in index: {any} {any}", .{ account_ref, curr_ref });
-        }
-        while (account_ref.next_ptr) |next_ref| {
-            // sanity check in debug mode
-            if (@import("builtin").mode == .Debug and next_ref.slot == account_ref.slot) {
-                std.debug.panic("duplicate slot in index: {any} {any}", .{ account_ref, next_ref });
-            }
```
Comment (Sobeston) [inline src/accountsdb/index.zig:240]: Change looks good, but is this debug check no longer useful?

**Review by Sobeston — CHANGES_REQUESTED** [review]

File: src/accountsdb/index.zig (hunk 1 above)
Comment (0xNineteen) [reply src/accountsdb/index.zig:240]: nah it was prev debugging code which i forgot to remove

**Review by Sobeston — APPROVED** [review]
:+1:

**Pull request merged**